src/csdx.ts

```diff
@@ -0,0 +1,360 @@
+          name: ["-c", "--config"],
+          description: "[optional] path of the config",
+          isOptional: true,
+          args: {
+            name: "config",
+          },
+        },
+        {
+          name: ["-k", "--stack-api-key"],
+          description: "API key of the source stack",
+          isOptional: true,
+          args: {
+            name: "stack-api-key",
+          },
+        },
+        {
+          name: ["-a", "--alias"],
+          description:
+            "The management token of the source stack from which you will export content",
+          isOptional: true,
+          args: {
+            name: "alias",
+          },
+        },
+        {
+          name: ["-d", "--data-dir"],
+          description:
+            "The path or the location in your file system to store the exported content. For e.g., ./content",
+          isOptional: true,
+          args: {
+            name: "data-dir",
+          },
+        },
+        {
+          name: "--branch",
+          description:
+            "The name of the branch where you want to export your content. If you don’t mention the branch name, then by default the content will be exported from all the branches of your stack",
+          isOptional: true,
+          args: {
+            name: "branch",
+          },
+        },
+        {
+          name: "--module",
+          description:
+            "[optional] Specific module name. If not specified, the export command will export all the modules to the stack. The available modules are assets, content-types, entries, environments, extensions, global-fields, labels, locales, webhooks, and workflows",
+          isOptional: true,
+          args: {
+            name: "module",
+            suggestions: [
+              "assets",
+              "content-types",
+              "entries",
+              "environments",
+              "extensions",
+              "global-fields",
+              "labels",
+              "locales",
+              "webhooks",
+              "workflows",
+            ],
+          },
+        },
+        {
+          name: "--content-types",
+          description: "[optional] content type",
+          isOptional: true,
+          args: {
+            name: "content-types",
+          },
+        },
+        {
+          name: "--secured-assets",
+          description: "[optional] use when assets are secured",
+          isOptional: true,
+          args: {
+            name: "secured-assets",
+          },
+        },
+      ],
+    },
+    {
+      name: "logout",
+      description:
+        "The auth:logout command lets you log out of Contentstack and clear the session",
+    },
+    {
+      name: "whoami",
+      description:
+        "The csdx auth:whoami command returns the username (email address) of the user who is currently logged in to the session",
+    },
+  ],
+  options: [
+    {
+      name: ["--help", "-h"],
+      description: "Show help for csdx",
+    },
```

Comment:
Makes --help show on each subcommand
```suggestion
    {
      name: ["--help", "-h"],
      description: "Show help for csdx",
      isPersistent: true,
    },
```